goalie/function_data.py

```diff
@@ -280,6 +280,53 @@ def _export_h5(self, output_fpath, export_field_types, initial_condition=None):
                             f = self._data[field][field_type][i][j]
                             outfile.save_function(f, name=name, idx=j)
 
+    def transfer(self, target, method="interpolate"):
+        """
+        Interpolate or project all functions from this :class:`~FunctionData` object to
+        the target :class:`~FunctionData` object.
+
+        :arg target: the target :class:`~FunctionData` object to which to transfer the
+            data
+        :type target: :class:`.FunctionData`
```

Comment:
Do the refs need to be `~.FunctionData`?

Reply:
Ah yes, sorry! I couldn't test this because of https://github.com/mesh-adaptation/mesh-adaptation-docs/issues/71 . So maybe we could do that before merging this?

Reply:
I think we can merge this first and address that later.

Reply:
Fixed in https://github.com/mesh-adaptation/goalie/pull/243/commits/87ff9783e15b5becf8ac97ef49ce50cba83aedb1